src/nodes/VariableDeclarationStatement.js

```diff
@@ -1,29 +1,43 @@
 const {
   doc: {
-    builders: { concat, join }
+    builders: { concat, group, indent, join, line, softline }
   }
 } = require('prettier/standalone');
 
+const embraceVariables = (doc, embrace) =>
+  embrace ? concat(['(', doc, ')']) : doc;
+
+const variables = (node, path, print) =>
+  group(
```

Comment:
I wonder if we can (and if we should) make a helper for this pattern, since we use it a lot. I guess something like this:

```
const printList = (node, path, print, key, separator = ',') => 
  group(
    concat([
      indent(
        concat([
          softline,
          join(
            concat([separator, line]),
            path.map(statementPath => print(statementPath), key)
          )
        ])
      ),
      softline
    ])
  );
```

(with better names for the function and the parameters).

Reply:
This is a tricky one.
The pattern is used a lot but with small differences.
Sometimes times is a `softline` and sometimes is a `line`, the separator and the content also changes I’ll see how it looks having a function with many variables in one go vs a step by step declaration.
 The system is already modular and there aren’t many files bigger than a screen.
I’ll give it a thought and play with the idea a little but probably for another PR.

Reply:
Sure. Just for the record: I'm fine with merging this as it is, I was just thinking out loud.

Reply:
Me too, I already got over my phase of having the driest possible code without really thinking of maintainability